### ISSUE TITLE
When installing blocked by 3rd-party mods, the Curseforge popup checks if "site connection is secure" forever.

### ISSUE THREAD

**Issue by Jamelos** [killpowa/Twickt-Launcher#1500]
Morning folks.
I explained the bug in the title

To reproduce it you just need to install a modpack with 3rd-party mods blocked.

**What should happen:**
The popup quickly check your site connection security.

- OS: Linux (Zorin Os Core)
- Launcher Version: Most Recent
- Java Version: Automatically downloaded by launcher
- Minecraft Version: Any
- Mod Loader: Any (But this happened on Fabric)

Thank you.

**Comment by Eskaan** [comment]
Duplicate of #1367
